code/modules/mining/mine_turfs.dm

```diff
@@ -628,7 +628,7 @@
 	for(var/thing in contents)
 		if(droppable(thing))
 			. = 1
-			drop(thing)
+			addtimer(src, "drop", 0, FALSE, thing)
```

Comment:
It means all the dropping is done at once instead of waiting for each item to finish; see `/turf/open/chasm/straight_down/lava_land_surface/drop(atom/movable/AM)`, which contains some sleep()s should would otherwise cause newly produced chasms to drop each item one at a time instead of all at once.